### DISCUSSION TITLE
Implement pause/suspend feature on Android(allow to work as bg process)

### DISCUSSION THREAD

**Discussion by andrey6498** [ppy/osu#15433] [Ideas] [answered]
So basically if you minimize osu lazer on android it just closes instantly, even if you just peek into task history etc, while most other android apps(even games) can be suspended properly and then resumed without resetting.

**Comment by bdach** [comment, marked as answer]
Known issue: https://github.com/ppy/osu/issues/6593#issuecomment-546041485

I had a very simple go at changing osuTK to use `GLSurfaceView` and calling `GLSurfaceView.setPreserveEGLContextOnPause` but it only worked reliably on one of the two devices I tested on for reasons currently unknown. Maybe I'll look at it again soon(TM) but no promises.

**Reply by frenzibyte** [reply]
For reference, this is tracked at https://github.com/ppy/osu-framework/issues/4200.
